src/library/scala/math/Ordering.scala

```diff
@@ -454,26 +454,26 @@ object Ordering extends LowPriorityOrderingImplicits {
     implicit object IeeeOrdering extends IeeeOrdering
   }
   @migration(
-    "  The new ordering does not affect the sorting, placing NaN at the end.\n" +
-    "  However, methods such as `lt`, `min`, and `equiv` now match `compare`\n" +
-    "  instead of having IEEE 754 behavior for -0.0F and NaN.\n" +
-    "  Import Ordering.Float.IeeeOrdering to retain the previous behavior.\n" +
-    "  See also https://www.scala-lang.org/api/current/scala/math/Ordering$$Float$.html", "2.13.0")
+    "  The default implicit ordering for floats now maintains consistency\n" +
+    "  between its `compare` method and its `lt`, `min`, `equiv`, etc., methods,\n" +
+    "  at the cost of nonconforming to IEEE 754's behavior for -0.0F and NaN.\n" +
+    "  Import Ordering.Float.IeeeOrdering to recover the previous behavior.\n" +
+    "  See also https://www.scala-lang.org/api/current/scala/math/Ordering$$Float$.html.", "2.13.0")
```

Comment:
I think the first thing we should clarify is that sort ordering is actually unaffected between 2.12 and 2.13.
Also "at the cost of" sounds like a value judgement implying IeeeOrdering is _better_, but it's difficult to say if it is in all situations.

Reply:
It didn't mean it as any judgement of "better".  Just that it's a cost/reward thing, not a 1 size fits all.

And I now understand what the "The new ordering does not affect the sorting, placing NaN at the end" bit means now, with your comments.  I dropped it as... it seemed unaffected, and so irrelevant for the migration notes.